bootstrap/src/lib/util.ts

```diff
@@ -1,23 +1,33 @@
-const { v4: uuidv4 } = require('uuid')
+import {
+  Execute,
+  ExecuteSync,
+  AdapterRequest,
+  AdapterResponse,
+  WrappedAdapterResponse,
+} from '@chainlink/types'
+import { v4 as uuidv4 } from 'uuid'
 
-const isObject = (o) => o !== null && typeof o === 'object' && Array.isArray(o) === false
+export const isObject = (o: unknown) =>
+  o !== null && typeof o === 'object' && Array.isArray(o) === false
 
-const parseBool = (value) => {
+export const parseBool = (value: any) => {
   if (!value) return false
   const _val = value.toString().toLowerCase()
   return (_val === 'true' || _val === 'false') && _val === 'true'
 }
 
 // We generate an UUID per instance
-const uuid = () => {
+export const uuid = () => {
   if (!process.env.UUID) process.env.UUID = uuidv4()
   return process.env.UUID
 }
 
-const toAsync = (execute, data) =>
-  new Promise((resolve) => execute(data, (statusCode, data) => resolve({ statusCode, data })))
+export const toAsync = (execute: ExecuteSync, data: AdapterRequest) =>
+  new Promise<WrappedAdapterResponse>((resolve) =>
+    execute(data, (statusCode: number, data: AdapterResponse) => resolve(data)),
```

Comment:
Something looks off here, shouldn't we `resolve({ statusCode, data })`?

```suggestion
    execute(data, (statusCode: number, data: AdapterResponse) => resolve({ statusCode, data })),
```